src/website/views/components/icon.pug

```diff
@@ -95,36 +95,54 @@ p.-text Two alternate small classes are available for specific app scenarios. Us
 h3 Colors
 p.-text By default, icons are the same color as text. Use Chi text color utilities to set a specific color.
 .example.-mb--3
-  .-p--3
-    .a-grid.-align-items--end
-      each color in ['base', 'primary', 'success', 'warning', 'danger', 'inverse', 'muted']
-        div.-p--2
-          div(class=`a-icon -md -text--${color}`)
+  .a-grid.-noGutter
+    .a-col.-w--12.-d--flex.-flex--row.-justify-content--center.-flex--wrap
+      each type in ['text--body', 'text--primary', 'text--success', 'text--warning', 'text--danger', 'text--muted']
+        div.-p--1.-p-sm--2.-text--center
+          div(class=`a-icon -md -${type}`)
             svg
               use(xlink:href='#icon-logo-centurylink')
+    .a-col.-w--12.-d--flex.-flex--row.-justify-content--center.-flex--wrap(style="background:#000;")
```

Comment:
@davidojedacl can you delete (style="background:#000;") and add -bg--black class?

Reply:
done